### ISSUE TITLE
Aligning far-off attr-pairs or self-closing tag

### ISSUE THREAD

**Issue by BurnzZ** [BurnzZ/xml-aligner#1]
Current aligning behavior takes note only the furthest attr-pair or self-closing tag. Extra space is produced if one of the the aforementioned in the group is too far away:

Example (self-closing tag would align to the one with **xyz**):

```
<a href = "asd"  />
<a href = "xyz"          />
<a href = "qwe" />
```

---

Some characters are not included in the regex line. Some of these include: `' / % * , = ;`

---

~~Commented tags are included in the logic for arrangement. (must ignore them)~~
_(I think this can be made into a feature instead of a bug, since commented tags blends naturally in the block and thus quite easier to read)_

**Comment by BurnzZ** [comment]
Partial fix: [278c735](https://github.com/BurnzZ/xml-aligner/commit/278c7352028b49bd4d7f244edfb536fb75eac816)
- leading spaces
- unmatched punctuation marks

**Comment by BurnzZ** [comment]
Additional fix: [b20f055](https://github.com/BurnzZ/xml-aligner/commit/b20f055bee9e434e6420e3212bbb40330065cc92)
- prevents removal of arbitrary spaces (greedy cleaning)

**Comment by BurnzZ** [comment]
Closing for now as first two points were solved. Still taking to note the last one.
